src/components/mods/MOptions.ts

```diff
@@ -0,0 +1,39 @@
+export class MOptions {
```

Comment:
Both storing reading in the mod configuration gives this class a whole lot of responsibility. I think that's a violation of the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). Let's keep the two separate: a mod should be able to list its options in one object, and reading them in should be done separately.

Reply:
In fact, I don't think reading in those options should even be part of this repository. That would belong in ModAttachr and/or UserWrappr.